### PR TITLE
Feature/add process received enmeshed onboarding package description

### PR DESCRIPTION
**Pull request by britsta**
Add scenario documentation for "Process received enmeshed onboarding package and create relationship" scenario

### CODE REVIEW

File: _data/navigation.yml [hunk 1]
```diff
@@ -85,7 +85,10 @@ docs_integrate:
       #  url: /integrate/event-introduction
       - title: "Use Cases"
         url: /integrate/use-cases
-
+  - title: Get in touch with other Identities
+    children:
+      - title: "Process received enmeshed onboarding package and create relationship"
```
Comment (jkoenig134) [inline _data/navigation.yml:90]: I encounter the wording "onboarding package" the first time now. Is this the RelationshipTemplate's content?

File: _includes/scenarios/integrate/process-received-enmeshed-onboarding-package-and-create-relationship.md [hunk 2]
```diff
@@ -0,0 +1,402 @@
...
+    "content": {
+      //Content of the RelationshipTemplate
+      ...
+    },
+    "truncatedReference": "<truncated reference of RelationshipTemplate>",
+    "maxNumberOfAllocations": <maximum number of allocations>,
+    "secretKey": "<secret key of RelationshipTemplate>"
+  }
+}
+``` --->
```
Comment (jkoenig134) [inline _includes/scenarios/integrate/process-received-enmeshed-onboarding-package-and-create-relationship.md:30]: many things commented out and some TODOs. should I wait with the review until this is sorted out?

Comment (britsta) [reply _includes/scenarios/integrate/process-received-enmeshed-onboarding-package-and-create-relationship.md]: I have removed the comments now (they were only leftovers from the creation process of the documentation), sorry for the confusion.

The only left TODO is now to insert a link to a guide that isn't published yet (but is an open PR (https://github.com/nmshd/documentation/pull/167) and requires a review too) as this documentation depends on other use cases and scenarios.

**Comment by britsta** [comment]
This PR is outdated and will be closed after the documentation for the "Establish a Relationship to another Identity" (https://github.com/nmshd/documentation/pull/194) scenario is published. 

The "Establish a Relationship to another Identity" scenario is a composition of the outdated scenario versions "Prepare enmeshed onboarding package" (https://github.com/nmshd/documentation/pull/167) and "Process received enmeshed onboarding package and create Relationship" (https://github.com/nmshd/documentation/pull/173).

**Pull request closed without merging**